web/src/components/Firebase/Firebase.js

```diff
@@ -1,7 +1,7 @@
 import { AuthProvider, useAuth } from '@redwoodjs/auth'
 import { RedwoodApolloProvider } from '@redwoodjs/web/apollo'
-import firebase from 'firebase/app'
-import 'firebase/auth'
+import { initializeApp, getApp, getApps } from 'firebase/app'
+import * as firebaseAuth from '@firebase/auth'
```

Comment:
Not sure about this, does this defeat the advantage of firebase v9 modular design to support tree-shaking? But the redwoodjs firebase authclient expects the redwood app to supply the module via client prop to AuthProvider